chunk/chunk_store.go

```diff
@@ -332,25 +339,45 @@ func (c *AWSStore) CreateTables() error {
 	return err
 }
 
-func bigBuckets(from, through model.Time) []int64 {
+func (c *AWSStore) bigBuckets(from, through model.Time) []string {
 	var (
 		secondsInHour = int64(time.Hour / time.Second)
 		fromHour      = from.Unix() / secondsInHour
 		throughHour   = through.Unix() / secondsInHour
-		result        []int64
+
+		secondsInDay = int64(24 * time.Hour / time.Second)
+		fromDay      = from.Unix() / secondsInDay
+		throughDay   = through.Unix() / secondsInDay
+
+		firstDailyBucket = c.dailyBucketsFrom.Unix() / secondsInDay
```

Comment:
My comment to @juliusv was the exact opposite - mainly because of whats coming in #181.